kalite/contentload/management/commands/channels/khan.py

```diff
@@ -1,284 +0,0 @@
-import base
```

Comment:
Sorry sorry, they were in one commit, so I would've had to cherry-pick your commit and then **ADD** code! Oh the horror of adding legacy code :)